burnman/eos/cork.py

```diff
@@ -131,6 +131,11 @@ def validate_parameters(self, params):
         Check for existence and validity of the parameters
         """
 
+        if 'T_0' not in params:
+            params['T_0'] = 298.15
+        if 'P_0' not in params:
+            params['P_0'] = 0.
```

Comment:
It looks like you are changing the default reference pressure from 10^5 Pa to 0 Pa.  Is this intended?